Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM golang:1.11.4 as builder
-WORKDIR /go/src/github.com/dechiad1/danny-ac/
+WORKDIR /go/src/github.com/aserhat/repo-whitelist-controller
```

Comment:
hm. if we use go modules i think we dont need a gopath like such

Reply:
Definitely up to changing this part in a next update.